pkg/webhook/validator.go

```diff
@@ -131,6 +154,7 @@ func (v *Validator) ValidatePodSpecable(ctx context.Context, wp *duckv1.WithPod)
 	// Attach the spec for down the line to be attached if it's required by
```

Comment:
You can ignore it, but you could include the object meta to this comment.

Reply:
Yup, done!